x-pack/plugins/apm/ftr_e2e/cypress.json

```diff
@@ -12,5 +12,8 @@
   "viewportHeight": 900,
   "viewportWidth": 1440,
   "video": false,
-  "screenshotOnRunFailure": false
+  "screenshotOnRunFailure": false,
+  "retries": {
```

Comment:
This will reduce the number of flaky test reported in the CI, also used  by [fleet](https://github.com/elastic/kibana/blob/e046687d11949c6ef6850dff6c40880bed6ac0c0/x-pack/plugins/fleet/cypress/cypress.json#L10), [security_solution](https://github.com/elastic/kibana/blob/e046687d11949c6ef6850dff6c40880bed6ac0c0/x-pack/plugins/security_solution/cypress/cypress.json#L8) and [enterprise_search](https://github.com/elastic/kibana/blob/1c4c81521303210d10fb0e93e06152d588690ea8/x-pack/plugins/enterprise_search/public/applications/app_search/cypress.json#L5)